frontend/web/components/pages/onboarding/OnboardingConnectPanel/OnboardingConnectPanel.scss

```
@@ -47,10 +47,10 @@
       border: 0;
       border-radius: 0;
       color: var(--color-text-default);
-      // Reserve the scrollbar gutter permanently so hover doesn't shove the
-      // layout (the global dark-scroll mixin toggles display:none on a
-      // space-reserving webkit bar otherwise).
-      overflow-x: scroll;
+      // Scroll only when the snippet overflows. When it does, the webkit rule
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)